pkg/options/options.go

```diff
@@ -37,6 +37,10 @@ type Options struct {
 	TelemetryHost                        string
 	Collectors                           CollectorSet
 	Namespaces                           NamespaceList
+	Shard                                int32
+	TotalShards                          int
+	Pod                                  string
+	Namespace                            string
```

Comment:
Let's rename this to `PodNamespace`. Since we already have `Namespaces`, `PodNamespace` would be a bit more clearer to the uninformed reader.